include/nbl/ui/CWindowManagerWin32.h

```diff
@@ -1,5 +1,6 @@
 #ifndef C_WINDOWMANAGER_WIN32
 #define C_WINDOWMANAGER_WIN32
+#ifdef _NBL_PLATFORM_WINDOWS_
```

Comment:
IWindowManager.h `#include` must be above this (or anything else that includes `compile_config.h` unconditionally

Reply:
important

Reply:
above NBL_PLATFORM_WINDOWS

Reply:
@sadiuk TODO

Reply:
fixed it myself